.github/workflows/flow-macos.yml

```diff
@@ -7,26 +7,28 @@ on:
   workflow_call:
     inputs:
       test-config:
-        description: 'Test configuration environment variable (e.g. "CONFIG=concurrent_write" or "QUICK=1")'
+        description: 'Test configuration environment variable (e.g. "CONFIG=tls" or "QUICK=1")'
         type: string
       coordinator:
         type: boolean
         default: true
       standalone:
         type: boolean
         default: true
+      redis-ref:
+        type: string
 
 jobs:
   build-macos:
     strategy:
       fail-fast: false
       matrix:
-        arch: [macos-11, macos-13-xlarge] # macos-13-xlarge has M1 chip
-    uses: ./.github/workflows/common-flow-host.yml
+        arch: [macos-11] #, macos-13-xlarge] # macos-13-xlarge has M1 chip
```

Comment:
Remind me why did you remove `macos-13` ?

Reply:
Because it's not available for the free tier.

Reply:
Let's decide what's the perferable way of testing with M1 with @adamiBs  - WDYT?